prowler/lib/outputs/common_models.py

```diff
@@ -75,3 +80,37 @@ class FindingOutput(BaseModel):
     related_to: str
     notes: str
     prowler_version: str = prowler_version
+
+
+class Output(ABC):
+    _data: object
```

Comment:
What if this output holds all the findings?

Reply:
Good point!